.eslintrc

```diff
@@ -2,6 +2,11 @@
   "parserOptions": {
     "project": "./tsconfig.json"
   },
+  "plugins": [
+    "react",
+    "react-native",
+    "react-hooks"
+  ],
   "extends": [
     "standard-with-typescript",
     "standard-jsx"
```

Comment:
need to extend the plugins you added?

- plugin:react-native/all
- plugin:react-hooks/all

Reply:
let me check